deploy-service/common/src/main/java/com/pinterest/deployservice/handler/CommonHandler.java

```diff
@@ -448,6 +454,20 @@ void internalTransition(String deployId, EnvironBean envBean) throws Exception {
 
         if (shouldSendFinishMessage(state, newPartialDeployBean.getState(), deployBean.getSuc_date())) {
             jobPool.submit(new FinishNotifyJob(envBean, deployBean, newPartialDeployBean, deployBean.getBuild_id()));
+
+            if (newPartialDeployBean.getState() == DeployState.SUCCEEDING) {
+                //Set SucDate to avoid resent SUCCEEDING
+                newPartialDeployBean.setSuc_date(System.currentTimeMillis());
+            }
+
+            // Submit post-webhooks, if exists
+            EnvWebHookBean
```

Comment:
Did you mean to include this change in this PR?

Reply:
Removed